draft-ietf-gnap-core-protocol.md

```diff
@@ -2736,8 +2736,8 @@ release subject claims, the response could look like this:
     },
     "subject": {
         "sub_ids": [ {
-           "subject_type": "email",
-           "email": "user@example.com",
+           "subject_type": "opaque",
+           "id": "J2G8G8O4AZ",
```

Comment:
No need to have `,` in `"id": "J2G8G8O4AZ",`.

Reply:
thanks, this is fixed